storage/tests/unit/test_blob.py

```diff
@@ -366,7 +366,7 @@ def test__make_transport(self, fake_session_factory):
 
     def test__get_download_url_with_media_link(self):
         blob_name = 'something.txt'
-        bucket = mock.Mock(spec=[])
+        bucket = _Bucket(name='IRRELEVANT')
```

Comment:
It is possible to set up a mock once and use it in multiple tests if that is really appropriate:

Set it up in `setUp` and assign it as a class instance, either restoring things in `tearDown` or just passing the mock and `with`-ing it in the tests.